proxy/proxy.go

```diff
@@ -148,8 +147,9 @@ func (proxy *Proxy) ListenAndServe() error {
 func (proxy *Proxy) weaveCIDRsFromConfig(config *docker.Config, hostConfig *docker.HostConfig) ([]string, error) {
 	if hostConfig != nil &&
 		hostConfig.NetworkMode != "" &&
+		hostConfig.NetworkMode != "default" &&
```

Comment:
@mavenugo right, that's what I thought. So it's the Docker API and CLI docs that are wrong then.